PyInstaller/hooks/hook-appdirs.py

```diff
@@ -0,0 +1,19 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2013-2017, PyInstaller Development Team.
+#
+# Distributed under the terms of the GNU General Public License with exception
+# for distributing bootloader.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+
+
+"""
+Import hook for appdirs.
+
+On Windows, appdirs tries 2 different methods to get well-known directories
+from the system: First with win32com, then with ctypes. Not using win32com here
+might avoid including Python's COM support altogether.
```

Comment:
Well, I struggled with this sentence a long time - and it seems I picked a better sentence when writing the pull request later ;)